src/Loading/Loading.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import loadingImg from '../images/loading.png';
+
+export default function Loading() {
+  return (
+    <div className='post-overlay'>
+      <img
+        className='spin'
+        alt='Updating the Post'
+        src={loadingImg}
+        style={{
+              "height": "10em",
+              "width": "auto",
+              "margin": "auto",
+              "padding": "4em"
+        }}>
```

Comment:
This is just nice to embed the styles, because now wherever I import this component I don't have to worry about targetting it It just simplifies the reasoning about it.